src/TeamSpeak3QueryApi/Query/QueryCommand.cs

```diff
@@ -1,7 +1,8 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using TeamSpeak3QueryApi.Net.Parameters;
 
-namespace TeamSpeak3QueryApi.Net
+namespace TeamSpeak3QueryApi.Net.Query
```

Comment:
This is part of the query client and belongs to the `TeamSpeak3QueryApi.Net` namespace.

Reply:
Cause a query command is still part of the query client. So it should stay in the query folder.

Reply:
The namespcae layout is as follows:

`namespace TeamSpeak3QueryApi.Net`: Everything related to the Query protocol itself (that also the regular TS client uses)
`namespace TeamSpeak3QueryApi.Net.Specialised`: While the name `Specialised` is disputable, this contains the concrete server query implementation.

One core principle is that the child namespaces should only rely on the parent namespaces (not some sibling namespace) if possible. Introducing a `Query` namespace would break this semantic.

Reply:
You mean I should change the class name QueryClient to just Client cause it´s in the namespace Query? (For example)